bin/tx-prover/prometheus.yml

```diff
@@ -0,0 +1,15 @@
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+# A scrape configuration containing exactly one endpoint to scrape:
+scrape_configs:
+  # The job name is a label that is used to group targets in the Prometheus UI.
+  # It can be any string.
+  - job_name: "proxy"
```

Comment:
Since this is an arbitrary name - should we pick something else? Maybe something like "tx_prover" would be a more descriptive name then "proxy"?